server/src/fastTagsSymbolProvider.ts

```diff
@@ -0,0 +1,103 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Cody Hoover. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+import * as cp from 'child_process';
+import {
+    SymbolKind, SymbolInformation, DocumentSymbolParams,
+    Range, Position, Location, Files as VscodeFiles
+} from 'vscode-languageserver';
+import { ISymbolProvider, ILogger } from './ghcModInterfaces';
+
+import { Files } from './utils/files';
+
+export class FastTagsSymbolProvider implements ISymbolProvider {
+
+    private executable: string;
+    private workspaceRoot: string;
+    private logger: ILogger
+
+    public constructor(executable: string, workspaceRoot: string, logger: ILogger) {
+        this.executable = executable;
+        this.workspaceRoot = workspaceRoot;
+        this.logger = logger;
+    }
+
+    public getSymbolsForFile(documentSymbolParams : DocumentSymbolParams): Thenable<SymbolInformation[]> {
+        let uri = documentSymbolParams.textDocument.uri;
+        let command = `${this.executable} -o - .${VscodeFiles.uriToFilePath(uri)}`;
```

Comment:
I'm testing on windows and the `uri` comes as an absolute path.
So it will be something like `C:\...` and then a dot gets appended to it. If I remove the dot it works perfectly :)